applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -129,15 +132,19 @@ static void le_audio_event_publish(enum le_audio_evt_type event, struct bt_conn
 static void cap_start_worker(struct k_work *work)
 {
 	int ret;
-	uint8_t device_index;
+	struct stream_index stream_idx;
 
 	/* Check msgq for a pending start procedure */
-	ret = k_msgq_get(&cap_start_msgq, &device_index, K_NO_WAIT);
+	ret = k_msgq_get(&cap_start_msgq, &stream_idx, K_NO_WAIT);
 	if (ret) {
 		LOG_ERR("Failed to get device index for pending cap start procedure: %d", ret);
 		return;
 	}
 
+	uint8_t lvl1_idx = stream_idx.lvl1_idx;
```

Comment:
I see this in multiple places. If we need to shorten, then we need to reconsider the naming of the `struct stream_index`. It does say "_idx" in two places which is not required.

Reply:
Let's discuss